core/application/impl/app_configuration_impl.cpp

```diff
@@ -159,6 +159,7 @@ namespace kagome::application {
         purge_wavm_cache_(def_purge_wavm_cache_),
         offchain_worker_mode_{def_offchain_worker_mode},
         enable_offchain_indexing_{def_enable_offchain_indexing},
+        subcommand_chain_info_{false},
```

Comment:
Why not def_subcommand_chain_info, for consistency?

Reply:
Should `is_telemetry_enabled_(true)` (few lines up) be changed too?